packages/third-parties/src/google/ga.tsx

```diff
@@ -54,14 +54,14 @@ export function GoogleAnalytics(props: GAParams) {
   )
 }
 
-export function sendGAEvent(..._args: Object[]) {
+export function sendGAEvent(eventName: string, eventParameters: {[key: string]: string | number}) {
```

Comment:
I think this type is too restrictive.

Some gtag events take an `item` parameter, which is an array of Item objects.

https://developers.google.com/tag-platform/gtagjs/reference/events

Reply:
Thank you for your review!

The url you attached return 500 status. Is that url about gtag ```event``` command?

I can't find that ```items``` parameters in recent docs about gtag.
https://developers.google.com/tag-platform/gtagjs/reference#event

Reply:
I added type GA events include ```items``` parameters you referenced.
Can you check this changes?

Reply:
I think your types won't allow custom events. Maybe there is an existing official set of types from Google that we could use?

Reply:
The type allow custom events. And I also added types from Google docs.

And if user input types from Google official types, user can get autocomplete  Google official params.

Also user can input custom types.
I pushed new commit. Please check that.

Reply:
Thank you for your maintaining. I know you're busy, but could you please check one more time, this commit would be very helpful.